src/components/tabWizard.tsx

```diff
@@ -0,0 +1,130 @@
+import _ from 'lodash/fp';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Clickable } from 'src/components/common';
+import { HorizontalNavigation } from 'src/components/keyboard-nav';
+import { terraSpecial } from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+
+const styles = {
+  tabWizard: {
+    container: {
+      display: 'flex',
+      alignItems: 'center',
+      fontWeight: 400,
+      textTransform: 'uppercase',
+      height: '2.25rem',
+      borderBottom: `1px solid ${terraSpecial()}`,
+      flex: '',
+    },
+    tab: {
+      flex: 'none',
+      padding: '0 1em',
+      marginBottom: '8px',
+      height: '100%',
+      alignSelf: 'stretch',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      borderBottomWidth: 8,
+      borderBottomStyle: 'solid',
+      borderBottomColor: 'transparent',
+    },
+    active: {
+      borderBottomColor: terraSpecial(),
+      borderBottomWidth: 12,
+      fontWeight: 600,
+    },
+    progressBar: {
+      height: '4px',
+      backgroundColor: `${terraSpecial()}`,
+      transition: 'width 0.3s ease-in-out',
+    },
+  },
+};
+
+/**
+ * Creates the primary tab bar for workspaces and workflows.
+ * Semantically, this is actually a menu of links rather than true tabs.
```

Comment:
The tab bar looks like I should be able to click on each tab and navigate into it, but that is not working for me on the PR site.